build/debian-base/Makefile

```diff
@@ -18,7 +18,7 @@ REGISTRY ?= staging-k8s.gcr.io
 IMAGE ?= $(REGISTRY)/debian-base
 BUILD_IMAGE ?= debian-build
 
-TAG ?= 0.4.1
+TAG ?= 1.0.0
```

Comment:
nit: many/most other components are tagging "v1.0.0" - note the leading "v"

Reply:
done